api/base/settings/defaults.py

```diff
@@ -61,7 +61,8 @@
 SESSION_COOKIE_SECURE = osf_settings.SECURE_MODE
 SESSION_COOKIE_HTTPONLY = osf_settings.SESSION_COOKIE_HTTPONLY
 SESSION_COOKIE_SAMESITE = osf_settings.SESSION_COOKIE_SAMESITE
-SESSION_ENGINE = 'django.contrib.sessions.backends.db'
+SESSION_ENGINE = 'django.contrib.sessions.backends.cache'
```

Comment:
* A few thoughts for this one:
  * Probably comment one option out so that it is easier to switch between them
  * Put a comment to explain how to and which one to use
  * Maybe update `local.py` as well?

Reply:
I want to make redis as our default session storage BE. So I won't update `defualts.py`. However I'll add something to `local-dist.py` to indicate the possibility of switching between BEs.